### PR TITLE
Deletions

### PR DESCRIPTION
**Pull request by jetstream0**

### CODE REVIEW

**Comment by jetstream0** [comment]
please no delete branch thanks

**Comment by Codemonkey51** [comment]
Hey @jetstream0 it should prob send all entry content (using embeds to a deletion channel) not just first 140 kthx

**Comment by Codemonkey51** [comment]
Also the raw_entry content (except history)

File: server.py [hunk 1]
```diff
@@ -343,7 +343,13 @@ async def edit_entry_post(request):
 @routes.post('/delete')
 async def delete_entry(request):
 	#404 until I actually implement this - rediar/prussia/jetstream
-	raise web.HTTPNotFound()
+	if not request.is_admin:
+		return web.HTTPFound('/')
+	entry_id = request.query.get('id')
+	entry_data = await database.get_entry(entry_id)
+	if not entry_data:
+		return "hey man there's no entry data, get your grip together!"
```
Comment (mat-1) [inline server.py:351]: Use single quotes, not double quotes (escape the apostrophe in the text)

File: server.py [hunk 2]
```diff
@@ -343,7 +343,13 @@ async def edit_entry_post(request):
 @routes.post('/delete')
 async def delete_entry(request):
 	#404 until I actually implement this - rediar/prussia/jetstream
```
Comment (mat-1) [inline server.py:345]: you implemented it now, so this comment should be removed

File: discordbot.py [hunk 3]
```diff
@@ -38,7 +38,7 @@ async def log_edit(editor, title, time):
 async def log_delete(title, time, content):
 	channel = client.get_channel(770468181486600253)
 	await channel.send(f'{title} has been deleted (through Repldex [direct database deletions are not detected]) at {time}')
-	await channel.send(f'First 140 characters of entry: `{content[:140]}`'
+	await channel.send(f'First 140 characters of entry: `{content[:140]}`')
```
Comment (mat-1) [inline discordbot.py:41]: This should link the entry url (using its id)

File: templates/edit.html [hunk 4]
```diff
@@ -52,8 +52,16 @@
...
 			{% endif %}
-      <br>
+		<br>
 			<button>Submit</button>
 		</form>
+		{% if is_admin %}
+			<button onclick="delete()">Delete Entry</button>
+			<script>
+				function delete() {
+					alert("this does not work yet")
```
Comment (mat-1) [inline templates/edit.html:62]: lol, what? don't merge if it doesn't work

**Pull request merged**